apps/nextjs-website/react-components/components/common/Common.tsx

```diff
@@ -5,9 +5,11 @@ import { CtaButtonProps } from '../../types/common/Common.types';
 export const CtaButtons = ({
   ctaButtons,
   theme,
+  disableRipple = false,
 }: {
   ctaButtons: ReadonlyArray<CtaButtonProps | JSX.Element>;
   theme: 'dark' | 'light';
```

Comment:
```suggestion
  theme?: 'dark' | 'light';
```
Following suggestion above.